### ISSUE TITLE
Training a mixed Chinese and English timbre model

### ISSUE THREAD

**Issue by pipijiev12** [FunAudioLLM/CosyVoice#615]
Currently, I use the example/libritts/cosyvoice/run.sh script to train the model. I found that libritts is an all-English dataset. If I need to train a new mixed Chinese and English voice model, can I just add the Chinese and English datasets of the corresponding characters to libritts for training?

**Comment by aluminumbox** [comment]
yes you can, you need mixed CH/EN data, not just pure CH and pure EN data

**Comment by pipijiev12** [comment]
> yes you can, you need mixed CH/EN data, not just pure CH and pure EN data

Does the dataset format require special handling?
